commit 24a03ff8021e9f2f2d65bf1b0f1aae2f4d61cb08
Author: John Levon <john.levon@joyent.com>
Date:   2019-08-08T16:15:01+00:00 (9 weeks ago)
    
    OS-7930 lx_boot_zone_docker was checked in after OS-7685
    Reviewed by: Robert Mustacchi <rm@joyent.com>

diff --git a/usr/src/lib/brand/lx/zone/lx_boot_zone_docker b/usr/src/lib/brand/lx/zone/lx_boot_zone_docker
deleted file mode 100755
index 9d332a723d..0000000000
--- a/usr/src/lib/brand/lx/zone/lx_boot_zone_docker
+++ /dev/null
@@ -1,17 +0,0 @@
-#!/bin/ksh -p
-#
-# This file and its contents are supplied under the terms of the
-# Common Development and Distribution License ("CDDL"), version 1.0.
-# You may only use this file in accordance with the terms of version
-# 1.0 of the CDDL.
-#
-# A full copy of the text of the CDDL should have accompanied this
-# source.  A copy of the CDDL is also available via the Internet at
-# http://www.illumos.org/license/CDDL.
-#
-
-#
-# Copyright 2014 Joyent, Inc.  All rights reserved.
-#
-
-# Hand control back to lx_boot
